fiche.c

```diff
@@ -613,12 +606,7 @@ static void *handle_connection(void *args) {
             print_error("Couldn't generate a valid slug!");
             print_separator();
 
-            // Cleanup
-            free(c);
-            free(slug);
-            close(c->socket);
```

Comment:
NOTE: This was broken before moving the cleanup to the `exit` label because it called `free(c)` before dereferencing `c`.